app/nexus.go

```diff
@@ -352,18 +366,20 @@ func NewNexusCommand(getNexusClientFn GetNexusClientFn) (CommandOut, error) {
 								"Only the fields that are provided will be updated",
 							Flags: []cli.Flag{
 								endpointNameFlag,
+								endpointDescriptionOptionalFlag,
```

Comment:
do we need to update description field for command?

Reply:
Updated description.

Reply:
OS cli doesn't have an additional comment for the description field.
https://github.com/bergundy/cli/blob/187b82da3e3d8702df476bc418abf9b706f7f95a/temporalcli/commands.gen.go#L852

I think we can leave it for now.